pkg/controller/nodenetworkstateconfiguration/nodenetworkstateconfiguration_controller.go

```diff
@@ -200,14 +201,19 @@ func (r *ReconcileNodeNetworkStateConfiguration) Reconcile(request reconcile.Req
 
 	nmstateOutput, err := nmstate.ApplyDesiredState(instance)
 	if err != nil {
-		errmsg := fmt.Errorf("error reconciling nodenetworkstate at desired state apply: %v", err)
+		errmsg := fmt.Errorf("error reconciling nodenetworkstate at desired state apply: %s, %v", nmstateOutput, err)
 
 		retryErr := r.setCondition(false, errmsg.Error(), request.NamespacedName)
 		if retryErr != nil {
 			reqLogger.Error(retryErr, "Failing condition update failed while reporting error: %v", errmsg)
 		}
 
-		return reconcile.Result{}, errmsg
+		if strings.Contains(errmsg.Error(), "rollback cause:") {
```

Comment:
Why only on rollback? We don't want to re-reconcile even if it failed validation of the scheme. Or even if it succeeded with rollback, we should not get into a loop of rollbacking and reconfigurations. Please tell me if I'm missing something. Is there a case where we would want to retry the configuration before user manually edits the Policy?

Reply:
Let's fail always, I don't find a failure scenario where a re-concile would fix it.

Reply:
Then please also add the "manual intervention" message to the condition.